packages/webpack-config-single-spa/lib/webpack-config-single-spa.js

```diff
@@ -48,8 +48,8 @@ function webpackConfigSingleSpa(opts) {
           },
         },
         {
-          test: /\.css$/,
-          include: /node_modules/,
+          test: /\.css$/i,
+          include: [/node_modules/, /src/],
```

Comment:
This is needed in order to be able to import local css files.

Reply:
👍